docs/assemblies/proc_hotfixing-the-data-plane-rpm-content.adoc

```diff
@@ -0,0 +1,42 @@
+[id="proc_hotfixing-the-data-plane-rpm-content-{context}"]
+= Hotfixing the data plane RPM content
+
+[role="_abstract"]
+
+RPM hotfix content is installed directly on to the data plane nodes.
+
+.Procedure
+
+. Obtain the RPM hotfix content from the source and store it locally.
++
+----
+$ mkdir -p <hotfix_id>/rpms
+$ cp /path/to/hotfix/*.rpm <hotfix_id>/rpms
+----
++
+* Replace `<hotfix_id>` with a hotfix identifier such as a Jira issue, for example `osprh-0000`
+
+. Copy the RPM hotfix content to the affected data plane nodes.
++
+----
+$ ssh <ssh_user>@<data_plane_node> mkdir -p /tmp/<hotfix_id>/rpms
+$ scp <hotfix_id>/rpms/*.rpm <ssh_user>@<data_plane_node>:/tmp/<hotfix_id>/rpms
+----
++
+* Replace `<hotfix_id>` with a hotfix identifier such as a Jira issue, for example `osprh-0000`
```

Comment:
Move the <hotfix_id> replace statement to after the <data_plane_node> statement, so that the user-replaceable variables are listed in the same order as they appear in the commands.

Reply:
Done.